nvidia/nvidia.go

```diff
@@ -104,19 +97,14 @@ func NewNvidiaInfo(ctx context.Context, runtime string) (types.GPUManager, error
 }
 
 func isGPUInstance(ctx context.Context) (bool, error) {
-	// Only check if we're on a GPU instance type
-	sess := session.Must(session.NewSession())
-	metadatasvc := ec2metadata.New(sess)
-	instanceType, err := metadatasvc.GetMetadata("instance-type")
-	if err != nil {
+	if _, err := os.Stat(nvidiaDev); err != nil {
+		if os.IsNotExist(err) {
+			logger.G(ctx).Info("Not on a GPU instance type. No GPU info available.")
```

Comment:
I know this was there before, but I don't think we need to log this. This is the "normal" path.

Reply:
Good call - moved so that it logs in the "abnormal" case.